internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -86,6 +90,14 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 		return errors.Errorf("invalid AWS AMI Image %s", cluster.ProvisionerMetadataKops.AMI)
 	}
 
+	cncVPCName := fmt.Sprintf("mattermost-cloud-%s-command-control", provisioner.environment)
+	cncVPCCIDR, err := awsClient.GetCIDRByVPCTag(cncVPCName, logger)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get the CIDR for the VPC Name %s", cncVPCName)
+	}
```

Comment:
Does this work in the dev account? I talked to @stylianosrigas and he says there is no VPC name in the dev account that matches this name. He says there is one called `mattermost-cloud-dev`.

Reply:
We should add this tag name in dev 
@stylianosrigas wdyt?

Reply:
yep confirmed, in dev is called `mattermost-cloud-dev` or we change in dev or add the if here

Reply:
update in dev to follow the same pattern